src/magic/magic.hpp

```diff
@@ -73,6 +73,8 @@ static const int SPELL_DASH = 51;
 static const int SPELL_SELF_POLYMORPH = 52;
 static const int SPELL_9 = 53;
 static const int SPELL_10 = 54;
+static const int SPELL_CRAB_FORM = 55;
```

Comment:
Would be better to set these to higher numbers we'll never reach e.g 10000, 10001. New spells will require these changing as NUM_SPELLS increases as these aren't real spells. Though does this need to be sequential to fetch the item's icon?

Reply:
these are strictly used to get the item icon index. so they need to be sequential.

Reply:
these are not real spells - just numbers with a name

Reply:
Yep just use spell 9 and 10 indices and we'll skip em for next spells

Reply:
done